lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1804,22 +1816,22 @@ namespace {
                                  Identifier className, CFPointeeInfo info,
                                  EffectiveClangContext effectiveContext);
 
-    /// Mark the given declaration as the Swift 2 variant of a Swift 3
-    /// declaration with the given name.
-    void markAsSwift2Variant(Decl *decl, ImportedName swift3Name,
-                             DeclContext *newDC = nullptr) {
-      ASTContext &ctx = decl->getASTContext();
+    /// Mark the given declaration as an older Swift version variant of the
+    /// current name.
+    void markAsOlderVariant(Decl *decl, ImportedName correctSwiftName) {
+      // TODO: some versions should be deprecated instead of unavailable
```

Comment:
@jrose-apple Should be named `markAsVariant` instead, since we might be using this on newer names with older requested version?